ci/run_integration_test.sh

```diff
@@ -222,9 +254,11 @@ main() {
     log INFO "$(docker compose version)"
 
     log INFO "Building Cuebot image..."
-    docker build -t opencue/cuebot -f cuebot/Dockerfile . &>"${TEST_LOGS}/docker-build-cuebot.log"
+#    docker build -t opencue/cuebot -f cuebot/Dockerfile . &>"${TEST_LOGS}/docker-build-cuebot.log"
```

Comment:
Is this an accidental change?

Reply:
Yup, sorry about that.
Fixed with latest merge.